### PR TITLE
[FIX][14.0]Dependencies in view

### PR DESCRIPTION
**Pull request by flotho**
running -u all cause an issue :
```
wip-test odoo.addons.base.models.ir_ui_view: delete_last_line n'est pas une action valide sur base.automation

View name: Automations
Error context:
 view: ir.ui.view(1145,)
 xmlid: view_base_automation_form
 view.model: base.automation
 view.parent: ir.ui.view(15,)
 file: /home/florent.thomas/DEV/Odoo_Server_And_Libs/Servers_14.0/OCB/addons/base_automation/views/base_automation_view.xml
 
2022-08-19 10:29:00,179 2679017 WARNING thesocialitefamily14-wip-test odoo.modules.loading: Transient module states were reset 
2022-08-19 10:29:00,184 2679017 ERROR thesocialitefamily14-wip-test odoo.modules.registry: Failed to load registry 
```

### CODE REVIEW

**Comment by flotho** [comment]
ping @legalsylvain

File: server_action_navigate/__manifest__.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "category": "Tools",
     "website": "https://github.com/OCA/server-backend",
     "license": "AGPL-3",
-    "depends": ["base"],
+    "depends": ["base", "base_automation"],
```
Comment (legalsylvain) [inline server_action_navigate/__manifest__.py]: Nitpicking : you can remove base dependency.

**Review by legalsylvain — APPROVED** [review]
Thanks !

**Comment by flotho** [comment]
done @legalsylvain

**Comment by legalsylvain** [comment]
fast tracking trivial patch.

/ocabot merge patch

**Pull request merged**